easybuild/framework/easyblock.py

```diff
@@ -214,6 +214,18 @@ def _init_log(self):
 
         self.log = fancylogger.getLogger(name=self.__class__.__name__, fname=False)
 
+        # deprecate self.log.error in easyblocks, self.log.fatal should be used instead
+        def error(msg, *args, **kwargs):
```

Comment:
@JensTimmerman: `error` will still be there, yes, but not with the intended behaviour anymore since it's no longer redefined in `EasyBuildLog`

anyway, this PR will change significantly after some discussion with @stdweird, hold on...